packages/ra-i18n-i18next/README.md

```diff
@@ -0,0 +1,222 @@
+# i18next i18n provider for react-admin
+
+i18next i18n provider for [react-admin](https://github.com/marmelab/react-admin), the frontend framework for building admin applications on top of REST/GraphQL services. It relies on [i18next](https://www.i18next.com/).
+
```

Comment:
Perhaps you could say a few words about why one would prefer this package over the polyglot one?